google/resource_logging_sink.go

```diff
@@ -30,6 +31,38 @@ func resourceLoggingSinkSchema() map[string]*schema.Schema {
 			Description:      `The filter to apply when exporting logs. Only log entries that match the filter are exported.`,
 		},
 
+		"exclusions": {
+			Type:        schema.TypeList,
+			Optional:    true,
+			Computed:    true,
+			Description: `Log entries that match any of the exclusion filters will not be exported. If a log entry is matched by both filter and one of exclusion_filters it will not be exported.`,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"name": {
+						Type:        schema.TypeString,
+						Required:    true,
+						Description: `A client-assigned identifier, such as "load-balancer-exclusion". Identifiers are limited to 100 characters and can include only letters, digits, underscores, hyphens, and periods. First character has to be alphanumeric.`,
+					},
+					"description": {
+						Type:        schema.TypeString,
+						Optional:    true,
+						Description: `A description of this exclusion.`,
+					},
+					"filter": {
+						Type:        schema.TypeString,
+						Required:    true,
+						Description: `An advanced logs filter that matches the log entries to be excluded. By using the sample function, you can exclude less than 100% of the matching log entries`,
+					},
+					"disabled": {
+						Type:        schema.TypeString,
```

Comment:
Why is this a string but it seems to be a boolean in the API client? We could just use a TypeBool which should remove some of the conversions

Reply:
Absolutely, there was no reason why `disabled` had to be `TypeString`. I have changed it to `TypeBool`. 
Thanks for the catch!